core/oslc4j-jena-provider/src/test/java/org/eclipse/lyo/oslc4j/provider/jena/MessageBodyReaderTest.java

```diff
@@ -0,0 +1,65 @@
+package org.eclipse.lyo.oslc4j.provider.jena;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collection;
+
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.lyo.oslc4j.core.model.OslcMediaType;
+import org.eclipse.lyo.oslc4j.provider.jena.test.resources.ResourceWithoutShape;
+import org.eclipse.lyo.oslc4j.provider.jena.test.resources.TestResource;
+import org.junit.Test;
+
+public class MessageBodyReaderTest {
+
+    @Test
+    public void testOslcXmlProvider() {
+        OslcXmlProvider provider = new OslcXmlProvider();
+
+        boolean isReadable = provider.isReadable(TestResource.class, null, TestResource.class.getAnnotations(),
```

Comment:
@jhemm2 I am not sure the 3rd argument is passed correctly:

> if the message body is to be converted into a method parameter, this will be the annotations on that parameter returned by Method.getParameterAnnotations

from https://docs.oracle.com/javaee/7/api/javax/ws/rs/ext/MessageBodyReader.html

Reply:
Not sure, too as it also states:
_**annotations** an array of the annotations on the declaration of the artifact that will be initialized with the produced instance_

What do you think would be correct here?